### PR TITLE
index: add additional translation

### PR DESCRIPTION
**Pull request by JungMinu**
Add additional translation for korean index

### CODE REVIEW

**Comment by fhemberger** [comment]
Thanks!

@nodejs/iojs-ko Can you please review the changes?

File: locale/ko/index.md [hunk 1]
```diff
@@ -5,12 +5,12 @@ labels:
...
-  other-stable-downloads: Other Stable Downloads
-  stable: Stable
+  other-lts-downloads: 다른 LTS 다운로드
+  other-stable-downloads: 다른 안정 버전 다운로드
+  stable: 안정 버전
   lts: LTS
-  tagline-stable: Latest Features
-  tagline-lts: Mature and Dependable
+  tagline-stable: 최신 기능
+  tagline-lts: 안정화, 신뢰됨
```
Comment (yous) [inline locale/ko/index.md]: How about '안정적, 신뢰도 높음'?

**Comment by JungMinu** [comment]
@yous Thanks, I've updated

**Comment by outsideris** [comment]
@fhemberger We reviewed it.

**Pull request merged**